packages/core/helper-plugin/src/hooks/useRBAC.ts

```diff
@@ -67,8 +67,15 @@ export const useRBAC = (permissionsToCheck, passedPermissions) => {
           try {
             const {
               data: { data },
-            } = await post('/admin/permissions/check', {
-              permissions: matchingPermissions.map(({ action, subject }) => ({ action, subject })),
+            } = await post<
+              { data: { data: boolean[] } },
+              AxiosResponse<{ data: { data: boolean[] } }>,
+              { permissions: Permission[] }
```

Comment:
I don't know that the second argument is necessary but I'm not sure how to skip it...